ifsbench/env.py

```diff
@@ -0,0 +1,197 @@
+# (C) Copyright 2020- ECMWF.
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+# In applying this licence, ECMWF does not waive the privileges and immunities
+# granted to it by virtue of its status as an intergovernmental organisation
+# nor does it submit to any jurisdiction.
+
+from abc import ABC, abstractmethod
+from collections import UserList
+from enum import auto, Enum
+import os
+
+from ifsbench.logging import debug
+
+__all__ = ['EnvHandler', 'EnvOperation', 'EnvPipeline']
+
+
+class EnvOperation(Enum):
+    """
+    Enum of environment operations.
+
+    Specifies operations on environment variables.
+
+    Attributes
```

Comment:
Same issue - see correct Docstring for class attributes in previous comment or #18